### DISCUSSION TITLE
유니티 월드 작업시 로그수집

### DISCUSSION THREAD

**Discussion by soontae1** [naverz/zepeto-studio-kor#251] [5. Q&A]
특정 이벤트 발생시 firebase나 외부통신하여 로그 수집이 가능할까요?

**Comment by x4cf** [comment]
가이드 규정에는 외부통신이 금지라고 나와있습니다.
https://studio.zepeto.me/guides/world-guidelines
2.5.5를 참고해주세요

**Reply by soontae1** [reply]
외부링크로 연결이 아니라 단순 로그수집도 안되는건지 해서 질문남긴거였어요

**Reply by x4cf** [reply]
단순 로그수집이라고 TCP통신으로 정보를 주고받습니다.
외부링크 연결이라는건 꼭 인간의 언어로 적힌 URL주소가 아니어도 모든 주소를 포함하고있으리라 예상합니다.
